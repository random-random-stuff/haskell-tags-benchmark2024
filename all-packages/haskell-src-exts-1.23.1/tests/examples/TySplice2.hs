{-# LANGUAGE TemplateHaskell #-}
t1 :: $( [t|Int|] )
t1 = 1

