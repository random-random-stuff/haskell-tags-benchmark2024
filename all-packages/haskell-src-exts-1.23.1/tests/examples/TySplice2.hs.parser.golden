ParseOk
  ( Module
      SrcSpanInfo
        { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 1 1 5 1
        , srcInfoPoints =
            [ SrcSpan "tests/examples/TySplice2.hs" 1 1 1 1
            , SrcSpan "tests/examples/TySplice2.hs" 2 1 2 1
            , SrcSpan "tests/examples/TySplice2.hs" 2 1 2 1
            , SrcSpan "tests/examples/TySplice2.hs" 2 1 2 1
            , SrcSpan "tests/examples/TySplice2.hs" 3 1 3 1
            , SrcSpan "tests/examples/TySplice2.hs" 5 1 5 1
            , SrcSpan "tests/examples/TySplice2.hs" 5 1 5 1
            ]
        }
      Nothing
      [ LanguagePragma
          SrcSpanInfo
            { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 1 1 1 33
            , srcInfoPoints =
                [ SrcSpan "tests/examples/TySplice2.hs" 1 1 1 13
                , SrcSpan "tests/examples/TySplice2.hs" 1 30 1 33
                ]
            }
          [ Ident
              SrcSpanInfo
                { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 1 14 1 29
                , srcInfoPoints = []
                }
              "TemplateHaskell"
          ]
      ]
      []
      [ TypeSig
          SrcSpanInfo
            { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 2 1 2 20
            , srcInfoPoints = [ SrcSpan "tests/examples/TySplice2.hs" 2 4 2 6 ]
            }
          [ Ident
              SrcSpanInfo
                { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 2 1 2 3
                , srcInfoPoints = []
                }
              "t1"
          ]
          (TySplice
             SrcSpanInfo
               { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 2 7 2 20
               , srcInfoPoints =
                   [ SrcSpan "tests/examples/TySplice2.hs" 2 7 2 9
                   , SrcSpan "tests/examples/TySplice2.hs" 2 19 2 20
                   ]
               }
             (ParenSplice
                SrcSpanInfo
                  { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 2 7 2 20
                  , srcInfoPoints =
                      [ SrcSpan "tests/examples/TySplice2.hs" 2 7 2 9
                      , SrcSpan "tests/examples/TySplice2.hs" 2 19 2 20
                      ]
                  }
                (BracketExp
                   SrcSpanInfo
                     { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 2 10 2 18
                     , srcInfoPoints =
                         [ SrcSpan "tests/examples/TySplice2.hs" 2 10 2 13
                         , SrcSpan "tests/examples/TySplice2.hs" 2 16 2 18
                         ]
                     }
                   (TypeBracket
                      SrcSpanInfo
                        { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 2 10 2 18
                        , srcInfoPoints =
                            [ SrcSpan "tests/examples/TySplice2.hs" 2 10 2 13
                            , SrcSpan "tests/examples/TySplice2.hs" 2 16 2 18
                            ]
                        }
                      (TyCon
                         SrcSpanInfo
                           { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 2 13 2 16
                           , srcInfoPoints = []
                           }
                         (UnQual
                            SrcSpanInfo
                              { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 2 13 2 16
                              , srcInfoPoints = []
                              }
                            (Ident
                               SrcSpanInfo
                                 { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 2 13 2 16
                                 , srcInfoPoints = []
                                 }
                               "Int")))))))
      , PatBind
          SrcSpanInfo
            { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 3 1 3 7
            , srcInfoPoints = []
            }
          (PVar
             SrcSpanInfo
               { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 3 1 3 3
               , srcInfoPoints = []
               }
             (Ident
                SrcSpanInfo
                  { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 3 1 3 3
                  , srcInfoPoints = []
                  }
                "t1"))
          (UnGuardedRhs
             SrcSpanInfo
               { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 3 4 3 7
               , srcInfoPoints = [ SrcSpan "tests/examples/TySplice2.hs" 3 4 3 5 ]
               }
             (Lit
                SrcSpanInfo
                  { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 3 6 3 7
                  , srcInfoPoints = []
                  }
                (Int
                   SrcSpanInfo
                     { srcInfoSpan = SrcSpan "tests/examples/TySplice2.hs" 3 6 3 7
                     , srcInfoPoints = []
                     }
                   1
                   "1")))
          Nothing
      ]
  , []
  )
